NuGetResources/NuGetPackageFeed3.cs

```diff
@@ -0,0 +1,421 @@
+﻿//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+namespace Microsoft.PackageManagement.NuGetProvider
+{
+    using Microsoft.PackageManagement.NuGetProvider.Resources;
+    using Microsoft.PackageManagement.Provider.Utility;
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+
+    /// <summary>
+    /// Implements NuGet v3 package feed
+    /// </summary>
+    public class NuGetPackageFeed3 : NuGetRemoteFeedBase, INuGetPackageFeed
+    {
+        public NuGetPackageFeed3(NuGetServiceInfo primaryServiceEndpoint)
+        {
+            this.Endpoints.Add(primaryServiceEndpoint);
+        }
+
+        public NuGetSearchResult Find(NuGetSearchContext findContext, NuGetRequest request)
```

Comment:
How do we populate and install the dependencies of a package? Is it based on the .NET Fx and Standard version for the client machine/application?

Reply:
All dependencies are downloaded, regardless of .NET version installed on the machine (it's not really easily discoverable). This part hasn't changed much from v2 (see NugetLightClient.cs - GetPackageDependenciesToInstall

---
In reply to: [154749472](https://github.com/OneGet/NuGetProvider/pull/25#discussion_r154749472) [](ancestors = 154749472)

Reply:
For package dependency parsing, see PackageBaseConverter.cs

---
In reply to: [154794977](https://github.com/OneGet/NuGetProvider/pull/25#discussion_r154794977) [](ancestors = 154794977,154749472)